gsoc-2024-backend/src/main/java/backend/BackendVerticle.java

```diff
@@ -11,10 +17,46 @@ public static void main(String[] args) {
     vertx.deployVerticle(new BackendVerticle())
       .onFailure(Throwable::printStackTrace)
       .onSuccess(v -> System.out.println("Deployed backend"));
+
+  }
+  
+  public void getJoke(Promise<String > promise) {
+    WebClient client = WebClient.create(vertx);
```

Comment:
This will create a client (and a pool of connections) for each request. Can you provide a better solution?

Reply:
i will move it to start().thanks for info.